tests/helpers.js

```diff
@@ -79,21 +79,26 @@ class FuzzTest {
 	}
 
 	execute() {
-		if (this.jestTestFile) {
+		if (this.jestTestFile !== "") {
```

Comment:
If `jestTestFile` is `undefined` Jest would be used now.

Reply:
Not sure why I changed that! Reverted back now.